prac_07/project_managment.py

```diff
@@ -0,0 +1,63 @@
+"""
+CP1404 | project and project management | Liam Eime
+Estimate time = 1.5hrs
+Actual time =
+"""
+
+import datetime
+from prac_07.project import Project
+
+MENU = """- (L)oad projects
+- (S)ave projects
+- (D)isplay projects
+- (F)ilter projects by date
+- (A)dd new project
+- (U)pdate project
+- (Q)uit"""
+
```

Comment:
If you add `>>> ` as an extra line on the end of the MENU constant, then when you print menu and get their choice you can just use `choice = input(MENU).upper()`